Backend/app/rider-platform/rider-app/Main/src/Storage/Queries/RideExtra.hs

```diff
@@ -304,7 +306,22 @@ findAllByRiderIdAndRide (Id personId) mbLimit mbOffset mbOnlyActive mbBookingSta
       (Se.Desc BeamB.startTime)
       (Just limit')
       (Just offset')
-
+  otherActivePartyBooking <-
+    if isOnlyActive
```

Comment:
Can we change it to this to only call the query if booking is not there with riderId.
`if isOnlyActive && null bookings' then`

Reply:
✅